lang/LangSource/PyrObject.cpp

```diff
@@ -110,7 +110,7 @@ PyrSymbol *s_identitydictionary;
 PyrSymbol *s_linkedlist;
 PyrSymbol *s_sortedlist;
 PyrSymbol *s_array;
-PyrSymbol *s_list, *s_method, *s_fundef, *s_frame, *s_class;
+PyrSymbol *s_list, *s_rest, *s_method, *s_fundef, *s_frame, *s_class;
```

Comment:
Btw what is the difference between an intrinsic class and other classes? I know intrinsic classes are specially defined in c++, but why is IdentityDictionary intrinsic while Rest should not be? If it's that IDict primitives need to access the member variable slots, then that's true of Rest now as well... so it seems possible to me that Rest could be intrinsic. But I don't know if there are other distinguishing features.